src/main/java/io/kestra/plugin/notion/DeletePage.java

```diff
@@ -0,0 +1,138 @@
+package io.kestra.plugin.notion;
+
+import io.kestra.core.http.HttpRequest;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.slf4j.Logger;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Delete a Notion page",
+    description = "Archives (deletes) an existing Notion page by ID. In Notion, pages are archived rather than permanently deleted, moving them to the trash."
```

Comment:
In that case let's call the task Archive to be close to the actual action?

Reply:
Yes definitely makes sense, fixed by https://github.com/kestra-io/plugin-notion/pull/2/commits/1e7d7c3b7ae4864f92e2681a2524e655afd8fb39